src/github.rs

```diff
@@ -608,10 +610,32 @@ impl Issue {
             .strip_prefix("https://api.github.com")
             .expect("expected api host");
         let comments_url = format!("{}{comments_path}", client.api_url);
-        client
-            .send_req(client.post(&comments_url).json(&PostComment { body }))
+        let comment = client
+            .json(client.post(&comments_url).json(&PostComment { body }))
             .await
             .context("failed to post comment")?;
+        Ok(comment)
+    }
+
+    pub async fn hide_comment(
+        &self,
+        client: &GithubClient,
+        node_id: &str,
+        reason: &str,
+    ) -> anyhow::Result<()> {
+        client
+            .graphql_query(
+                "mutation($node_id: ID!, $reason: ReportedContentClassifiers!) {
+                    minimizeComment(input: {subjectId: $node_id, classifier: $reason}) {
+                        __typename
+                    }
+                }",
```

Comment:
This doesn't seem to work. It generates an error:

```
failed to hide previous merge commit comment

Caused by:
    error: Variable $reason of type ReportedContentClassifiers! was provided invalid value
```

The classifier is an [enum](https://docs.github.com/en/graphql/reference/enums#reportedcontentclassifiers) with specific values, but it looks like this is passing in an arbitrary string. I would suggest adding an enum for the classifiers.

Reply:
Good catch. I was wondering what values where being accepted, I guess I didn't look enough to see the enum.

Added the enum with only the permitted values.